test/Main.purs

```diff
@@ -0,0 +1,16 @@
+module Test.Main where
+
+import Prelude
+
+import Data.Maybe (Maybe(..))
+import Data.Time.Duration (Milliseconds(..))
+import Effect (Effect)
+import Effect.Aff (launchAff_)
+import Test.Spec.Discovery (discover)
+import Test.Spec.Reporter.Codewars (codewarsReporter)
+import Test.Spec.Runner (defaultConfig, runSpec')
+
+main :: Effect Unit
+main = launchAff_ $ discover ".+Spec" >>= runSpec' config [ codewarsReporter ]
+  where
+  config = defaultConfig { timeout = Just (Milliseconds 12000.0) }
```

Comment:
:information_source: I checked in tests and so on. If there is a reason this should be avoided, one option could also be to have two spago files - or to remove them again.